### DISCUSSION TITLE
Add font or styling to U+2610 (Ballot Box)

### DISCUSSION THREAD

**Discussion by rlaphoenix** [pradyunsg/furo#395] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/17136956/157234125-6432c982-9f33-4e35-bf16-90c73414e463.png)

As you can see, the U+2611 (Ballot Box with Check) is already styled. It seems to be the Segoe UI Emoji font that's doing so.
Sadly, the general style of U+2610 doesn't match, even with the same font stack set by Furo, nor can I figure out a font-based way to do so.

It would be really nice to at least have the spacing match, or some kind of alternative to using these like GitHub Markdown Flavor did with [Task lists](https://github.blog/2014-04-28-task-lists-in-all-markdown-documents/). I understand this problem may not directly be related to the Furo project and may be a discussion directly with ReST but I thought I'd bring it up.

For reference, this is how it's done with GitHub Markdown Flavor - Task Lists:

- [ ] Unchecked
- [X] Checked

**Comment by pradyunsg** [comment, marked as answer]
GitHub does not use the font glyphs, but instead inlines an `input` tag. I don't believe there's any way to get the behaviour that you're hoping for with just fonts.
